diff --git a/sites/all/modules/contrib/file_entity/file_entity.module b/sites/all/modules/contrib/file_entity/file_entity.module
index 3ca0bb7..fcc5644 100644
--- a/sites/all/modules/contrib/file_entity/file_entity.module
+++ b/sites/all/modules/contrib/file_entity/file_entity.module
@@ -2155,6 +2155,9 @@ function file_entity_upload_validators_pre_render($element) {
  * Implements hook_file_default_displays_alter() on behalf of image.module.
  */
 function image_file_default_displays_alter(&$file_displays) {
+  // @HACK: Temporary change: there is no need to override the default file displays if they are set up properly.
+  return;
+
   // Images should be displayed as unstyled images by default.
   if (isset($file_displays['image__default__file_field_file_default'])) {
     $file_displays['image__default__file_field_file_default']->status = FALSE;
