diff --git a/modules/user/user.install b/modules/user/user.install
index 4e1a3c2..c65bae6 100644
--- a/modules/user/user.install
+++ b/modules/user/user.install
@@ -126,11 +126,10 @@ function user_schema() {
     'description' => 'Stores user data.',
     'fields' => array(
       'uid' => array(
-        'type' => 'int',
+        'type' => 'serial',
         'unsigned' => TRUE,
         'not null' => TRUE,
         'description' => 'Primary Key: Unique user ID.',
-        'default' => 0,
       ),
       'name' => array(
         'type' => 'varchar',
