diff --git a/modules/user/user.admin.inc b/modules/user/user.admin.inc
index 6ca330b..ad250c7 100644
--- a/modules/user/user.admin.inc
+++ b/modules/user/user.admin.inc
@@ -172,7 +172,8 @@ function user_admin_account() {
   );
 
   $query = db_select('users', 'u');
-  $query->condition('u.uid', 0, '<>');
+  // Hide UID1 from the users listing.
+  $query->condition('u.uid', 1, '>');
   user_build_filter_query($query);
 
   $count_query = clone $query;
